src/components/side_panel/pivot/pivot_defer_update/pivot_defer_update.xml

```diff
@@ -1,25 +1,29 @@
 <templates>
   <t t-name="o-spreadsheet-PivotDeferUpdate">
-    <Section
-      class="'align-items-center border-top d-flex flex-row justify-content-between py-1 pivot-defer-update'">
-      <Checkbox
-        label="deferUpdatesLabel"
-        title="deferUpdatesTooltip"
-        value="props.deferUpdate"
-        onChange="(value) => props.toggleDeferUpdate(value)"
-      />
-      <div t-if="props.isDirty">
-        <i
-          class="btn btn-sm pe-0 fa fa-undo"
-          title="Discard all changes"
-          t-on-click="() => props.discard()"
+    <Section class="'border-top d-flex flex-column justify-content-between py-1'">
+      <t t-set="message">Consider using a dynamic pivot with =PIVOT(1)</t>
+      <ValidationMessages messages="[message]" msgType="'info'"/>
```

Comment:
![image](https://github.com/user-attachments/assets/be51465b-d3fc-4448-9d14-a8e7d00d4ada)

Style is messed up:
- the validation show as an error with no background 
- only "defer update" has a gray background, it should be the whole panel footer